internal/logging/logging.go

```
@@ -0,0 +1,261 @@
+	defer fileMutex.Unlock()
+
+	if serviceFile != nil {
+		err := serviceFile.Close()
+		serviceFile = nil
+		return err
+	}
+	return nil
+}
+
+func Cleanup() {
+	CloseServiceLogFile()
```

golangci-lint

Error return value is not checked

(errcheck)